keyboards/redragon/k630/info.json

```diff
@@ -0,0 +1,190 @@
+{
+    "keyboard_name": "K630",
+    "manufacturer": "Redragon",
+    "url": "https://www.redragonzone.com/products/redragon-k630-gaming-mechanical-keyboard",
+    "maintainer": "bausensi",
+    "bootloader": "sn32-dfu",
+    
+    "processor": "SN32F248BF",
+    "usb": {
+        "vid": "0x0C45",
+        "pid": "0x5004",
+        "device_version": "0.0.1",
+        "force_nkro": true
+    },
+
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+
+    "debounce": 3,
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": [ "A0",  "A1",  "A2",  "A3", "A4", "A5",  "A6",  "A7",  "A8", "A9", "A10", "A11", "A12", "A13" ],
+        "rows": [ "D11", "D10", "D9",  "D8", "D7" ]
+    },
+    
+    "tapping": {
+        "term": 150
+    },
```

Comment:
this should be needed when using tap dancing, but not a simple `LT` as you seem to be using in the default keymap ( afaik). If this is needed for a custom user keymap, you can nest a `info.json` with it in the keymap dir